src/models/MediaImage.ts

```diff
@@ -61,41 +61,18 @@ export class MediaImageL {
     // ⏸️     renameSync(this.absPath + '2', this.absPath)
     // ⏸️ }
 
-    /* XXX: This should only be a stop-gap for a custom solution that isn't hampered by the browser's security capabilities */
-    /** Uses browser clipboard API to copy the image to clipboard, will only copy as a PNG and will not include metadata. */
+    /** Uses electron clipboard API to copy the image to clipboard, will only copy as PNG. */
     copyToClipboard = () => {
-        createHTMLImage_fromURL(URL.createObjectURL(this.getAsBlob()))
+        this.st.electronUtils
+            .copyImageToClipboard({
+                format: this.extension.split('.').pop(),
+                buffer: this.getArrayBuffer(),
```

Comment:
This will serialize the buffer from the webview to the electron shell; it probably work, and may or may not be the right way to go.,Every MediaImageL now have a valid local path at creation time; we may want to just send the media image path instead and load the buffer from the shell itself - or not. Unsure.